src/network/index.ts

```diff
@@ -170,4 +172,12 @@ export class StacksMocknet extends StacksMainnet implements StacksNetwork {
   }
 }
 
+export class SyvireanMainnet extends StacksMainnet implements StacksNetwork {
```

Comment:
I don't think we should export other named Network options, but instead I'd love to see a change to the StacksNetwork class where this would be possible:

```ts
const syvitaNetwork = new StacksMainnet('syvita');
const hiroNetwork = new StacksMainnet('hiro');
// or
const network = new StacksMainnet({url: 'https://mainnet.syvita.org'});
```

Reply:
@0xAsteria let me know if you plan to implement these changes, else I will close this PR until there is time to make this more possible.